template/node-armhf/Dockerfile

```diff
@@ -1,30 +1,36 @@
-FROM arm32v6/alpine:3.6
+FROM arm32v6/node:10.15.0-alpine
```

Comment:
Does this work without the prefix? We may be able to use the multi arch images? Ask in thy ARM channel for someone to validate if you don't own an RPi.

Reply:
```
$ echo 'node:10.15.0-alpine' | faas invoke mquery
Manifest List: Yes
Supported platforms:
 - linux/amd64
 - linux/arm/v6
 - linux/arm64
 - linux/386
 - linux/ppc64le
 - linux/s390x
```

Reply:
Seems like we can remove the prefixes?